vault/activity_log_test.go

```diff
@@ -4274,13 +4274,15 @@ func TestActivityLog_processNewClients_delete(t *testing.T) {
 	mount := "mount"
 	namespace := "namespace"
 	clientID := "client-id"
-	run := func(t *testing.T, isNonEntity bool) {
+	run := func(t *testing.T, clientType string) {
 		t.Helper()
+		isNonEntity := clientType == entityActivityType || clientType == ACMEActivityType
```

Comment:
Should the first equality be `clientType == nonEntityTokenActivityType` instead?

Reply:
If that is true, it tells me that we could also add a test for the interaction of that field with the delete api.